docs/rst/how_to/arise_poc.rst

```diff
@@ -0,0 +1,978 @@
+
+**TimescaleDB**
+   It is a time series database built on PostgreSQL, designed to handle large-scale, time-stamped data efficiently. It offers powerful SQL support for managing time series data, with features like automatic data partitioning, real-time aggregation, and advanced query capabilities. More information can be found in `Documentation <https://docs.timescale.com/>`_ and in `GitHub <https://github.com/timescale/timescaledb>`_
+
+**Grafana**
+   It is an open-source analytics and visualization platform that allows users to visualize data stored in TimeScaleDB through customizable dashboards.
+   It supports data representation using charts, graphs, tables, and alerts. Besides TimeScaleDB, Grafana integrates with a wide range of data sources, including Prometheus, InfluxDB, MySQL, PostgreSQL, Elasticsearch, and more.
+   It is released under the Apache License 2.0. More information can be found in `Documentation <https://grafana.com/>`_ and in `GitHub <https://github.com/grafana/grafana>`_.
+
+Step-by-step Guide
+===================
+
+This section outlines the process for rapidly deploying a fully functional testbed that includes all relevant actors.
+
+Requirements
+------------
+
+- Docker (Version 26.1.5)
+- Docker-compose (Version 1.29+)
+- RAM 8GB minimum
+
+**Install docker and docker-compose** by following the instructions available on the official web site:
+
+- Docker: `here <https://docs.docker.com/install/linux/docker-ce/ubuntu/>`_
+- Docker-Compose: `here <https://docs.docker.com/compose/install/>`_
+
+**Verify Installation**: Run the following commands to check Docker and Docker Compose:
+
+.. code-block:: bash
+
+   docker --version
+   docker compose --version
+
+Step 1 - Clone the ARISE PoC
+-----------------------------
+
+Open a terminal and move into a folder in which to create the new folder containing the ARISE PoC components.
+
+Then run:
+
+.. code-block:: bash
+
+   git clone "https://github.com/Engineering-Research-and-Development/arise-poc.git"
+
+Step 2 - Configure the ARISE PoC
+---------------------------------
+
+Before launching the ARISE PoC, it is essential to edit the docker-compose.yaml file and configure IoT Agent OPC UA service to align with your specific OPC UA Server's specifications.
+
+.. code-block:: bash
+
+   cd arise-poc/
+   nano docker-compose.yaml
+
+Here are some environment variables that must be configured:
+
+- ``IOTA_CB_SERVICE`` allows to define where the OPCUA IoT Agent sends NGSI-LD context data;
+- ``IOTA_SERVICE`` defines the devices are registered;
+- ``IOTA_OPCUA_ENDPOINT`` is used in FIWARE IoT Agent for OPC UA to specify the OPC UA server endpoint.
+
+For a more complete description on how to configure the IoT Agent, go to `link <https://github.com/Engineering-Research-and-Development/iotagent-opcua/blob/master/docs/howto.md>`_.
```

Comment:
Adding another `_` to the end of the link will avoid the "duplicate link" warning in CI checks. Apply this to the rest of the duplicated links. 

```suggestion
For a more complete description on how to configure the IoT Agent, go to `link <https://github.com/Engineering-Research-and-Development/iotagent-opcua/blob/master/docs/howto.md>`__.
```